framework/include/variables/MooseVariableFE.h

```diff
@@ -324,7 +329,9 @@ class MooseVariableFE : public MooseVariableFEBase
   }
 
   const FieldVariableValue & uDot() { return _u_dot; }
+  const FieldVariableValue & uDotdot() { return _u_dotdot; }
```

Comment:
@friedmud What's your thought on always requesting uDotdot, etc.? If necessary we can add flags like _need_u_dotdot.